packages/builder/src/pages/builder/portal/apps/index.svelte

```diff
@@ -299,35 +357,19 @@
             ]}
           />
           <div class="desktop-search">
-            <Search placeholder="Search" bind:value={searchTerm} />
+            <Search quiet placeholder="Search" bind:value={searchTerm} />
           </div>
         </div>
-        <ActionGroup>
-          <ActionButton
-            on:click={() => (layout = "grid")}
-            selected={layout === "grid"}
-            quiet
-            icon="ClassicGridView"
-          />
-          <ActionButton
-            on:click={() => (layout = "table")}
-            selected={layout === "table"}
-            quiet
-            icon="ViewRow"
-          />
-        </ActionGroup>
       </div>
       <div class="mobile-search">
         <Search placeholder="Search" bind:value={searchTerm} />
       </div>
-      <div
-        class:appGrid={layout === "grid"}
-        class:appTable={layout === "table"}
-      >
+      <div class="appTable">
         {#each filteredApps as app (app.appId)}
           <svelte:component
-            this={layout === "grid" ? AppCard : AppRow}
+            this={AppRow}
```

Comment:
Since this is hardcoded now to just be one component, we can remove the `<svelte:compoennt>` tag and just render a `<AppRow ... />` directly.